test/NuGet.Core.Tests/NuGet.PackageManagement.Test/PackageDownloaderTests.cs

```diff
@@ -331,9 +331,14 @@ public async Task GetDownloadResourceResultAsync_V2_DownloadsPackage()
                 {
                     var targetPackageStream = downloadResult.PackageStream;
 
+                    // calculate contentHash to verify we got the package correctly.
+                    var packageArchiveReader = new PackageArchiveReader(targetPackageStream);
+                    var contentHash = packageArchiveReader.GetContentHash(new CancellationToken());
+
                     // Assert
-                    // jQuery.1.8.2 is of size 185476 bytes. Make sure the download is successful
-                    Assert.Equal(185476, targetPackageStream.Length);
+                    // jQuery.1.8.2 has the following contentHash:
+                    Assert.Equal("cXOJxYC6ccYDP5FW1iOXhZww+7CyKdpiJbkR0YZxILNJ2zvM4VsrqOVKNRHnIWF78IixUfo/cw7Hz4M70MUbGg==",
```

Comment:
`const`-ize this hash and reference the constant here, [here](https://github.com/NuGet/NuGet.Client/pull/2483/files#diff-ad56a19bbc3fafbb8344fb63277f8614R384) and [here](https://github.com/NuGet/NuGet.Client/pull/2483/files#diff-ad56a19bbc3fafbb8344fb63277f8614R639).